pages/verification/index.tsx

```diff
@@ -107,6 +114,7 @@ export default function Verification(props: PageProps) {
         sendAnalytics(BetterSocialEventTracking.VERIFICATION_SCREEN_OPEN)
 
         const anonTokenFromCookie = Cookies.get(ITokenEnum.anonymousToken)
+        console.log('anontoken from cookie', anonTokenFromCookie);
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

Logging sensitive information like tokens directly to the console can be a security risk, especially in a production environment. If this log is for debugging purposes, make sure to remove it before deploying or wrap it with a condition that checks the environment.

```diff
- console.log('anontoken from cookie', anonTokenFromCookie);
+ if (process.env.NODE_ENV === 'development') {
+     console.log('anontoken from cookie', anonTokenFromCookie);
+ }
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->